simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/CardCreatorContract.kt

```diff
@@ -41,9 +41,9 @@ class CardCreatorContract(
       USER_IDENTIFIER_KEY to input.userIdentifier
     )
 
-  override fun parseResult(resultCode: Int, intent: Intent?): Output? =
+  override fun parseResult(resultCode: Int, intent: Intent?): Output =
     if (intent == null || resultCode != Activity.RESULT_OK) {
-      null
+      Output("", "")
```

Comment:
This gets the code to compile but the implications of losing nullability here probably warrants a review of where/how this contract is used.